Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:2-slim
+
+RUN mkdir -p /opt/app/
+WORKDIR /opt/app/
+
+RUN ln -sf /usr/share/zoneinfo/America/New_York /etc/localtime
+
+ADD requirements.txt /opt/app/
+ADD test_requirements.txt /opt/app/
+
+RUN pip install -r /opt/app/requirements.txt
+RUN pip install -r /opt/app/test_requirements.txt
+
+ADD . /opt/app/
+
+RUN python setup.py test
+RUN python setup.py install
+
+RUN ln -sf /usr/share/zoneinfo/UTC  /etc/localtime
```

Comment:
The tests fail when run with the timezone set to UTC: https://github.com/Netuitive/netuitive-client-python/issues/9